data_uploader/test/test_netbox_connection.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from unittest import mock
+from unittest.mock import NonCallableMock, patch
+
+import data_uploader.netbox_api.netbox_connection
+from nose.tools import raises
+from data_uploader.exceptions.missing_mandatory_param_error import MissingMandatoryParamError
+from data_uploader.netbox_api.netbox_connection import NetboxApi
+
+
+class NetboxApiTests(unittest.TestCase):
+    @staticmethod
+    @patch('netbox_api.netbox_connection.api')
+    def test_api_object_create(mock_api):
+        """
+        Tests that we do get an API object an
+        """
+        mock_url = "test_netbox.rl.ac.uk"
```

Comment:
Use `example.com` URLs so theres:
- No chance of it ever connecting
- Prevents us writing code specific to these URLs
- Helps identify where we're using mocks or real system tests (if we add them in the future)